hs_core/models.py

```diff
@@ -3734,7 +3734,7 @@ def get_relation_version_res_url(self, rel_type):
         relation_meta_obj = self.metadata.relations.filter(type=rel_type).first()
         if relation_meta_obj is not None:
             # get the resource url from resource citation
-            version_res_url = relation_meta_obj.value.split(',')[-1]
+            version_res_url = re.search(r'(?P<url>https?://[^\s,]+)', relation_meta_obj.value).group("url")
```

Comment:
A couple of questions: (1) re.search() only matches URLs starting with https which will not work for local development environment, so I don't think hardcoding https in the search pattern is a good idea. I remember you can get the URL schema, https or http, from a Django setting. I don't remember the exact setting name out of the top of my head, but I think it should be easy to find it out. (2) Do we need to group url? Since the versioning chain is single obsolescence chain, i.e., a resource could only have one VersionOf relation and ReplacedBy relation, so the old code to get the URL might work without the need for this regular expression search? I might have missed something here, but want to bring it up just in case.

Reply:
@hyi all very good points, thank you for taking the time! You are correct that this modification is unnecessary given the other changes, so I just pushed a commit to revert this.